7.1/Dockerfile

```diff
@@ -10,7 +10,6 @@ RUN apt-get -qq update > /dev/null && DEBIAN_FRONTEND=noninteractive apt-get -qq
     ca-certificates \
     nginx \
     wget \
-    git \
```

Comment:
@jcrombez git is required by composer to clone repos when dist mode is nos available (when using dev-master for example)